types/react-native-snap-carousel/index.d.ts

```diff
@@ -180,11 +179,11 @@ export interface CarouselProps<T> extends React.Props<ScrollViewProps> {
     /**
      * Optional styles for Scrollview's global wrapper
      */
-    containerCustomStyle?: StyleProp<ScrollViewStyle>;
+    containerCustomStyle?: StyleProp<ViewStyle>;
```

Comment:
I’m a little confused, isn’t this limiting the available styles to a superset of the available scrollview styles?

Reply:
It's supposed to be a ViewStyle according to the source https://github.com/archriss/react-native-snap-carousel/blob/59b1d90f7e5ad23d4c4ced241ec15379e5d2fb6d/src/carousel/Carousel.js#L46

Reply:
Ah gotcha 👍